src/worker.cc

```diff
@@ -85,7 +85,7 @@ const std::unordered_map<unsigned, valhalla::valhalla_exception_t> error_codes{
     {151, {151, "Exceeded max time", 400, HTTP_400, OSRM_INVALID_VALUE, "too_large_time"}},
     {152, {152, "Exceeded max contours", 400, HTTP_400, OSRM_INVALID_VALUE, "too_many_contours"}},
     {153, {153, "Too many shape points", 400, HTTP_400, OSRM_INVALID_VALUE, "too_large_shape"}},
-    {154, {154, "Path distance exceeds the max distance limit", 400, HTTP_400, OSRM_DISTANCE_EXCEEDED, "too_large_distance"}},
+    {154, {154, "Path distance exceeds the max distance limit of " + extra + " meters", 400, HTTP_400, OSRM_DISTANCE_EXCEEDED, "too_large_distance"}},
```

Comment:
`extra` is nowhere defined that this map could see it.

Reply:
@nilsnolde I apologize for that, This is my first open-source project that I've ever tried so I'm not exactly sure how it goes other than the tutorials I watched. Just for future reference, in order to have a proper development setup to build the project would I follow the "building from source - Windows" section?

Reply:
Jep right. I wouldn’t recommend developing Valhalla on Win, the tests won’t build and MSVC doesn’t have the greatest error messages IMO. It’s possible (and I think at least one of us does Win at least quite often, but he’s a more than battle tested dev), but quite a bit harder. 

There’s nothing special about open source code and I’d encourage you to contribute, IMO it’s pretty rewarding. I’m just suggesting that Valhalla is maybe a bit too hard if you’re generally new to programming/C++.

Reply:
Okay I see, I first learned C++ a little less than a year ago so I do understand most concepts at a basic level in this project but not at an expert level. I think I definitely need to look through the files more to get a better understanding of how all the files work together because I see what you mean now about how extra is not accessible anywhere with the map, it's hard to catch anything though when I don't have it built on my end. Is Linux the recommended build platform? I think I may be able to figure out how to build it with the Windows wsl ubuntu. Anyways, I appreciate your time and responses and hope to contribute in the future. and again I apologize for putting in the pull request the second time after I still wasn't able to figure out how to build it on my end.

Reply:
Don’t worry about it! I started 2 years ago, it takes a while. If you want to use Linux (which is highly recommend for any dev other than C# or so), I’d at least do a dual boot on your machine. With WSL I can possibly foresee frustration to make it work with VS Code or CLion;)